src/Uno.UI/UI/Xaml/Controls/ListViewBase/NativeListViewBase.Android.cs

```diff
@@ -56,6 +60,12 @@ public NativeListViewBase() : base(ContextHelper.Current)
 			InitializeSnapHelper();
 
 			MotionEventSplittingEnabled = false;
+
+#if __ANDROID_28__
+			_shouldRecalibrateFlingVelocity = Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.P;
+#else
+			_shouldRecalibrateFlingVelocity = Android.OS.Build.VERSION.SdkInt >= 28;
```

Comment:
Would it still be necessary to check whether we are targeting `MonoAndroid90` or not? The problem was that `BuildVersionCodes.P` isn't defined when targeting `MonoAndroid80`.

Reply:
@jeromelaban @MatFillion

Reply:
we can probaly just use `_shouldRecalibrateFlingVelocity = Android.OS.Build.VERSION.SdkInt >= 28;`

Reply:
by reading the bug report, are we sure the fix is introduced with android Q ? we could say `Android.OS.Build.VERSION.SdkInt == 28` instead of >=28

Reply:
Yeah, we will have to wait until the fix is released to update that.